### PR TITLE
Olesia

### PR DESCRIPTION
**Pull request by IgorKorobchenko**
Updates from Olesia

### CODE REVIEW

File: tests/test_login.py [hunk 1]
```diff
@@ -4,14 +4,39 @@
...
+#
+#     driver.quit()
 
 o = webdriver.FirefoxOptions()
 o.headless = True
 driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()), options=o)
 driver.get('https://www.saucedemo.com/')
-
 def test_login_page():
+    print("Hello")
```
Comment (MathiasD27) [inline tests/test_login.py:39]: Please add "!" after Hello

**Review by MathiasD27 — COMMENTED** [review]
Please delete

**Comment by OlesiaZhenirovska** [comment]
@MathiasD27 
Ok

File: tests/test_login.py (hunk 1 above)
Comment (OlesiaZhenirovska) [reply tests/test_login.py:39]: @MathiasD27 
ok

**Pull request closed without merging**